extensions/positron-r/src/provider.ts

```diff
@@ -216,11 +221,12 @@ export async function makeMetadata(
 	digest.update(rVersion);
 	const runtimeId = digest.digest('hex').substring(0, 32);
 
-	// Save the R home path and binary path as extra data.
+	// Save the R home path, binary path and RScript path as extra data.
 	// Also, whether this R installation is the "current" R version.
 	const extraRuntimeData: RMetadataExtra = {
 		homepath: rInst.homepath,
 		binpath: rInst.binpath,
+		rScriptPath: rScriptPath,
```

Comment:
I think I was expecting that `scriptpath` ends up as part of the `RInstallation` itself.

That way:
- you just pass it through here with very little fanfare
- we can pull it from the `RInstallation` if we need to use it elsewhere
-  `validateMetadata()` can also perform its check on `scriptpath`

Reply:
IIRC the only way to get this extra metadata into the `LanguageRuntimeMetadata` is to put it in `RMetadatExtra`. That's something we had to create, in fact, in order to keep both the binpath and homepath available to ourselves later.

Reply:
These `RInstallation` instances don't last very long and are just used to instantiate `positron.LanguageRuntimeMetadata`.